src/main/java/uk/go/hm/icb/service/dvla/DVLAService.java

```diff
@@ -85,7 +85,7 @@ public ICBResponse buildSingleMatchResponse(ICBResponse.ICBResponseBuilder respo
                             "-",
                             "-",
                             matchDriverLicense(request, dlRecord),
-                            "-"
+                            "-", null
```

Comment:
Passing null and doing null check on the other side is something we avoid by the earlier code.

Reply:
as per reply to ICBMatch comment, single method may be more suitable as LEV records may always have a flag